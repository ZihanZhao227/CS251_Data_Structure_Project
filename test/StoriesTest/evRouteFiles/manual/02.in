3
0 0 0 CITY Start
1 50 50 RESTAURANT R1
2 100 0 CITY End
