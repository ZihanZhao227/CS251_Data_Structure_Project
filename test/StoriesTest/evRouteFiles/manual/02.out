0 2 50.000 -1.0
NO_PATH
